docs/apis/http/oidc/_index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "OIDC"
+date: 2023-10-10T00:00:00+00:00
+weight: 21
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/apis/http/idp
+geekdocFilePath: _index.md
+geekdocCollapseSection: true
+---
```

Comment:
Should it go under `apis` or `clients` section?
CC @mmattel

Reply:
For me, neither.
IMHO this should go as a new section named `OIDC` into: https://owncloud.dev/ocis/identity-provider/

Reply:
If I would be a developer looking for how to integrate my app with oCIS I would first look into  API->Authentication and expect there a description about how to get myself tokens etc.

Reply:
as just discussed in the meeting, let's put 
- the general overview (currently contained in `_index.md`) into https://owncloud.dev/ocis/identity-provider/OIDC
- the detail description with the exampled (currently contained in `flow.md`) into https://owncloud.dev/apis/http/authorization
- link both places together